bittensor/btlogging/loggingmachine.py

```diff
@@ -172,9 +173,23 @@ def _create_file_handler(self, logfile: str):
         file_handler.setLevel(stdlogging.TRACE)
         return file_handler
 
+    def enable_first_party_logger(self, logger_name: str):
```

Comment:
these functions aren't necessary, as the enabling/disabling logger logic is handled by the various state transitions (trace, debug, on, and disabled)